db/system_distributed_keyspace.cc

```diff
@@ -168,6 +200,15 @@ future<> system_distributed_keyspace::start() {
         return _mm.announce_new_keyspace(ksm, api::min_timestamp);
     });
 
+    co_await ignore_existing([this] {
+        auto ksm = keyspace_metadata::new_keyspace(
+                NAME_EVERYWHERE,
+                "org.apache.cassandra.locator.EverywhereStrategy",
+                {},
```

Comment:
Please split out the new keyspace. This one is important and deserves its own commit (maybe its own pull request). I'm in favor, but we need to consider the impact here.

Reply:
I will create a new PR after #8283 gets merged because I don't want to deal with rebase hell (I think there will be some conflicts).